include/ppx/tri_mesh.h

```diff
@@ -60,6 +60,19 @@ struct TriMeshVertexData
     float3 bitangent;
 };
 
+//! @struct TriMesh16VertexData16
+//!
+//!
+struct TriMeshVertexData16
+{
+    glm::u16vec3 position;
```

Comment:
i think to be consistent with the other vector types, it would be better to do something like `using half3 = glm::u16vec3`

Reply:
Done.